src/components/grid/Grid.stories.tsx

```diff
@@ -37,6 +61,86 @@ export const defaultContainer = (): React.ReactElement => (
   </GridContainer>
 )
 
+export const customContainer = (): React.ReactElement => (
+  <GridContainer<CustomGridContainerProps> asCustom={CustomGridContainer}>
+    <li>
+      <Grid row>
+        <Grid col={1}>{testContent}</Grid>
+        <Grid col={11}>{testContent}</Grid>
+      </Grid>
+    </li>
```

Comment:
I have some thoughts on this, but they're kinda hard to articulate, because the use case for doing this is still a bit weird to me.

There's nothing inherently _wrong_ with putting Grid components inside an `li` in this case, but it is basically a perfect opportunity to use the custom Grid component as a `li` anyways.

I see you have a story for the custom Grid item as well, so you have that covered. My thinking at least is to just combine the two into a single `customElements` story where both the container and the grid items are custom elements (container can be `ul` and grid can be `li` maybe?)

Reply:
I love this idea!